.github/actions/setup-env/action.yml

```
@@ -25,7 +25,7 @@ runs:
       uses: pdm-project/setup-pdm@v3
       with:
         python-version: ${{ inputs.python-version }}
-        version: "2.4.6"
+        version: "2.5.4"
```

```suggestion
        version: "2.5.5"
```
https://github.com/pdm-project/pdm/releases/tag/2.5.5